<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditTextColorDialog</class>
 <widget class="QDialog" name="EditTextColorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Scenario Text Colors</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
   <item>
    <widget class="QLabel" name="color_list_label">
     <property name="text">
      <string>Colors:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="controls_hbox" stretch="0,1,0">
     <item>
      <widget class="QLabel" name="selected_color_label">
       <property name="text">
        <string>Selected color:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="selected_color_index">
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="select_color_button">
       <property name="text">
        <string>Pick Color</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="color_list"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialog_button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
